R/plink2cape.R

```diff
@@ -140,7 +140,7 @@ plink2cape <- function(ped = "test.ped", map = "test.map", pheno = "test.pheno",
   write.table(final_table, out, quote = FALSE, sep = ",", row.names = FALSE, col.names = FALSE,
   na = "-")
   
-	cross_obj <- read_population(out)
+	cross_obj <- read_population(out, verbose = verbose)
```

Comment:
does this need to be changed to TRUE or FALSE?

Reply:
I think here we want the user to be able to control it, so up in the arguments for the function the user can set verbose. And then plink2cape will use that value when it calls read_population. So I think that one is fine. Thanks for going through this so carefully!